speech/audioGUI.m

```diff
@@ -130,9 +132,13 @@
             plot_params = get_plot_defaults;
         end
     end
-    
+
     % optionally overwrite figure position
     if ~isempty(figpos), plot_params.figpos = figpos; end
+
+    % default to show formants
+    plot_params.yes_show_formants = 1;
+
```

Comment:
Can we put this in get_plot_defaults instead? Not sure why this should be separated out here.

Reply:
Or actually, do we need it at all if it's set in wave_viewer's get_plot_params?

Reply:
Agreed we don't need this -- changed.